contrib/systemd/system/netavark-dhcp-proxy.service.in

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Netavark DHCP proxy service
+Requires=netavark-dhcp-proxy.socket
+After=netavark-dhcp-proxy.socket
+Documentation=
+StartLimitIntervalSec=0
+
+[Service]
+Delegate=true
+Type=exec
+KillMode=process
+Environment=LOGGING="--log-level=info"
+ExecStart=/usr/libexec/podman/netavark-proxy -a 30
```

Comment:
since you already use a `.in` you need to replace that part at install with the actual path. This should not be hard coded.
Also needs the corresponding install makefile target of course.